doc/classes/Camera2D.xml

```diff
@@ -369,10 +371,13 @@
 			The camera's offset, useful for looking around or camera shake animations.
 		</member>
 		<member name="rotating" type="bool" setter="set_rotating" getter="is_rotating">
+			If [code]true[/code] the camera can rotate during gameplay. Default value: [code]false[/code].
 		</member>
 		<member name="smoothing_enabled" type="bool" setter="set_enable_follow_smoothing" getter="is_follow_smoothing_enabled">
+			If [code]true[/code] the camera gradually changes its velocity vector. Default value: [code]false[/code].
```

Comment:
Hmm, this makes it sound like the camera just moves by itself.  How about:

    If [code]true[/code] the camera's velocity changes gradually, preventing sudden starts and stops.